src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Slider/Slider_Inside_ListViewHeader.xaml.cs

```diff
@@ -6,12 +6,8 @@ namespace Uno.UI.Samples.Content.UITests.Slider
 	[SampleControlInfoAttribute("Slider", "Slider_Inside_ListViewHeader")]
 	public sealed partial class Slider_Inside_ListViewHeader : UserControl
     {
-		double OverallPerformance = 0.0;
```

Comment:
You cannot remove it: it seems to be data bound in the view

Reply:
You're right.